src/karmen_frontend/cypress/integration/organizations/adding_spec.js

```diff
@@ -16,29 +16,37 @@ describe("Organizations: Adding", function () {
   });
 
   it("fails with no name", function () {
-    cy.get('button[type="submit"]').click();
-    cy.get("form").contains("Name is required");
+    cy.get("input#username").type(email);
```

Comment:
Login should be handled separately by running a direct HTTP REST API request rather then going through the login form over and over. See the recommendations on https://docs.cypress.io/guides/references/best-practices.html#Organizing-Tests-Logging-In-Controlling-State

Usually, this is best implemented using Cypress custom command: https://docs.cypress.io/api/cypress-api/custom-commands.html#Usage

Reply:
After internal dicussion, we've decided not to block approval on this one and instead forward this to the upcoming e2e test suite upgrade.